src/PrestaShopBundle/Resources/views/Admin/Exception/error.html.twig

```diff
@@ -0,0 +1,76 @@
+{#**
+ * 2007-2019 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% extends '::base.html.twig' %}
+
+{% block stylesheets %}
+  <link rel="stylesheet" href="{{ asset('themes/new-theme/public/theme.css') }}" />
+{% endblock %}
+
+{% block title %}
+  {{ 'Oops... looks like an unexpected error occurred'|trans({},'Admin.Notifications.Error' ) }}
+{% endblock %}
+
+{% block body %}
+  <div class="container">
+    <div class="row mt-5">
+      <div class="col">
+        <div class="card">
+          <div class="card-body text-center">
+            <img class="img-responsive" src="{{ asset('themes/new-theme/img/error/500.svg') }}">
```

Comment:
image does not go through webpack build process, thus it is not in `/public`, should we change that or keep as is?

Reply:
Do you mean than if you put it in `/public`, there is a webpack error ?

Reply:
Did you want to put it through webpack processing, but could not because of an issue, or did you want to avoid the compilation step for this process and ask if it's alright ?